### DISCUSSION TITLE
send logs to zipkin

### DISCUSSION THREAD

**Discussion by hamedprog** [open-telemetry/opentelemetry-cpp#1195] [Q&A] [answered]
hello friends
I want to send my project logs to zipkin. system logs and library logs.

what I must do?

**Comment by esigo** [comment, marked as answer]
Hi @hamedprog, thanks for reaching out to us. `Zipkin` (or other tracing exporters) is not the right exporter for this purpose. You need to pick one of the log exporters e.g. [otlp_http_log](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/examples/otlp/http_log_main.cc) or [otlp_grpc_log](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/examples/otlp/grpc_log_main.cc).

You may also add [events](https://github.com/open-telemetry/opentelemetry-cpp/blob/44795b6d21738a8478d620c6f4cde6f12ea5ccac/api/include/opentelemetry/trace/span.h#L59) with your logs as attribute to your spans and keep using `Zipkin`.
